rotkehlchen/tests/unit/test_inquirer.py

```diff
@@ -888,6 +888,21 @@ def test_recursion_in_inquirer(inquirer: Inquirer, globaldb: GlobalDBHandler):
     assert inquirer.find_usd_price(A_USDT) != ZERO
 
 
+@pytest.mark.parametrize('should_mock_current_price_queries', [False])
+def test_recursion_handling_in_inquirer(inquirer: Inquirer, globaldb: GlobalDBHandler):
+    """This is a regression test that checks for the handling of the recursion error in the inquirer."""  # noqa: E501
+    a_usdt = A_USDT.resolve_to_evm_token()
+    with globaldb.conn.write_ctx() as write_cursor:
+        write_cursor.execute(
+            'INSERT INTO price_history(from_asset, to_asset, source_type, price, timestamp) VALUES(?, ?, ?, ?, ?)',  # noqa: E501
+            (a_usdt.identifier, a_usdt.identifier, 'E', '5', ts_now()),
+        )
+
+    price, oracle = inquirer.find_usd_price_and_oracle(A_USDT)
+    assert oracle == CurrentPriceOracle.MANUALCURRENT
+    assert price != ZERO_PRICE
```

Comment:
What should this one be? 5? instead of `!=` do assert to what this is equal to since it is static